packages/database/exp-types/index.d.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '@firebase/app-types';
+
+export { initializeDatabase, getDatabase } from '../src/exp/Database';
+
+export interface DataSnapshot {
+  child(path: string): DataSnapshot;
+  exists(): boolean;
+  exportVal(): any;
+  forEach(action: (a: DataSnapshot) => boolean | void): boolean;
+  getPriority(): string | number | null;
+  hasChild(path: string): boolean;
+  hasChildren(): boolean;
+  key: string | null;
+  numChildren(): number;
+  ref: Reference;
+  toJSON(): object | null;
+  val(): any;
+}
+
+export interface Database {
+  app: FirebaseApp;
+  useEmulator(host: string, port: number): void;
+  goOffline(): void;
+  goOnline(): void;
+  ref(path?: string | Reference): Reference;
+  refFromURL(url: string): Reference;
+}
+
+export class FirebaseDatabase implements Database {
```

Comment:
Why do we export the class?

Reply:
It matches the existing types, but we can delete this export.